### PR TITLE
OCM-2532 : Adding deleteAssociatedResources parameter to account delete command

### PR DESCRIPTION
**Pull request by tirthct**
**Changed**
- Adding deleteAssociatedResources parameter to account delete command
- Updated readme

**Tested**
- Passing deleteAssociatedReosources to true should remove account along with its related resources : tested

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -368,13 +368,15 @@ Use the `account` subcommand to delete a specific account for the passed account
 The following flags are available for `delete account`:
 
 ```
---dry-run                    If false, deletes the account for the given account ID, defaults to true.
--h, --help                   help for delete
+--dry-run                       If false, deletes the account for the given account ID, defaults to true.
+--delete-associated-resources   If true, deletes the associated resources like registry credentials, role bindings, etc. along with the account.
```
Comment (cristianoveiga) [inline README.md:372]: It would be good to list all the resources that are affected by this. Do we know/ can we figure this information?

Comment (tirthct) [reply README.md]: Yes. Added all of them in readme

Comment (cristianoveiga) [reply README.md]: thanks!

**Comment by cristianoveiga** [comment]
LGTM

**Pull request merged**